Source/MonoGame.Extended/Maps/Tiled/TiledTileLayer.cs

```diff
@@ -80,19 +83,19 @@ public List<VertexPositionTexture> RenderVertices()
                 }
                 var vertices = new VertexPositionTexture[4];
                 vertices[0] = new VertexPositionTexture(
-                    new Vector3(tile.X * TileWidth - 1, tile.Y * TileHeight, Z),
+                    new Vector3(tile.X * TileWidth - 0.9f, tile.Y * TileHeight, Z),
```

Comment:
`0.5f` was still showing some gaps, so I changed to `1f`, but this offset was visible with some zoom-in, so I ended with `0.9f` that don't show the gaps and isn't visible. I forgot to change the offset on the projection matrix, I will do it now.